src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -266,6 +268,44 @@ export default {
   },
 };
 ```
+#### `addMiddleware` option
```

Comment:
Just noting that there was a merge comment since the Dev Overlay PR merged in the mean time. Should all be fixed now!